rules/optimise-your-favicon/rule.md

```diff
@@ -0,0 +1,48 @@
+---
+type: rule
+archivedreason: 
+title: Do you optimise your website favicon?
+guid: df8d34e2-89c9-4a97-be6c-70f473890bdd
+uri: optimise-your-favicon
+created: 2023-07-28T12:00:00.0000000Z
+authors:
+- title: Jack Reimers
+  url: https://ssw.com.au/people/jack-reimers
+related:
+- 
+---
+
+Favicons are commonplace in web development, but did you know your favicon may not be properly optimised for all devices?
```

Comment:
I prefer answering the question (rule title) in the intro, and then going into more detail after